modules/extensions/src/views/lite/components/Dropdown.jsx

```diff
@@ -45,10 +45,8 @@ export class Dropdown extends React.Component {
   }
 
   renderSelect(inKeyboard) {
-    const placeholder = this.props.store.intl.formatMessage({
-      id: 'composer.dropdownPlaceholder',
-      defaultMessage: 'Select a subject'
-    })
+    const placeholder = this.props.intl.locale === 'fr' ? 'Sélectionnez un choix' : 'Select a choice'
```

Comment:
hmmm what happens when we start supporting more languages ... I.E when someone from the community starts to translate the UI in its language ?

Reply:
tbh idk... right now it doesn't update properly and it's one of the issues found by the government so I went for a quick fix... there might be a better one that I'm not aware of

Reply:
We will need to adapt slightly translation handling in channel web. Right now, translations are handled by ui-shared, whihc is used by the studio and the admin panel. We can't use it in channel web for size constraints. So this will do for now, but we need to rethink translations soon for the webchat